.gitlab/binary_build/cws_injector.yml

```diff
@@ -0,0 +1,34 @@
+---
+.cws_injector-build_common:
+  stage: binary_build
+  needs: ["go_mod_tidy_check"]
+  script:
+    - inv check-go-version
+    - inv -e cws-injector.build
+    - $S3_CP_CMD $CI_PROJECT_DIR/$CWS_INJECTOR_BINARIES_DIR/cws-injector $S3_ARTIFACTS_URI/cws-injector.$ARCH
+
+cws_injector-build_amd64:
+  extends: .cws_injector-build_common
+  rules:
+    !reference [.on_tag_or_a7]
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/deb_x64$DATADOG_AGENT_BUILDIMAGES_SUFFIX:$DATADOG_AGENT_BUILDIMAGES
+  tags: ["runner:main"]
+  needs: ["go_mod_tidy_check", "go_deps"]
+  variables:
+    ARCH: amd64
+  before_script:
+    - source /root/.bashrc
+    - !reference [.retrieve_linux_go_deps]
+
+cws_injector-build_arm64:
+  extends: .cws_injector-build_common
+  rules:
+    !reference [.on_tag_or_a7_all_builds]
```

Comment:
just to double check, the intent is to have x86_64 builds on all pipelines targeting agent 7 & tags, and only on tags & full pipeline for aarch64 builds, correct?

Reply:
Same here, updated to `[.on_tag_or_a7]`. Thanks !